src/test/skript/tests/syntaxes/structures/StructParse.sk

```diff
@@ -0,0 +1,11 @@
+parse:
+  results: {StructParse::parse::*}
+  code:
+    parse:
+      results: "oops"
+      code:
+        on script load:
+          broadcast "hi"
+
+test "StructParse":
+  assert {StructParse::parse::*} contains """oops"" cannot be set to strings" with "StructParse error not found"
```

Comment:
It sounds cool but I really honestly cannot tell what on earth is going on in your test, I think some more explanation or documentation for the feature might help (even some javadoc would be pretty cool).

Reply:
It's indeed cool! He is using a nested StructParse with invalid store object "oops" that will throw an error and catching it for testing